modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -170,6 +172,12 @@ public class RocksDbKeyValueStorage implements KeyValueStorage {
     /** Snapshot manager. */
     private volatile RocksSnapshotManager snapshotManager;
 
+    /**
+     * Revision listener for recovery only. Notifies {@link MetaStorageManagerImpl} of revision update.
+     * Guarded by {@link #rwLock}.
+     */
+    private LongConsumer revisionListener;
```

Comment:
I think that the name is not quite suitable, it is better to indicate that this is a listener on recovery, plus it would be better to use the new interface instead of `LongConsumer`,

Reply:
Why not LongConsumer? It is a listener of long after all, no need to create an extra interface for such a small task